src/test/scala/uk/gov/nationalarchives/Steps.scala

```
@@ -0,0 +1,182 @@
+package uk.gov.nationalarchives
+
+import cats.effect.unsafe.implicits.global
+import cats.effect.{IO, Sync}
+import cats.implicits._
+import fs2.Stream
+import fs2.interop.reactivestreams._
+import io.circe.generic.auto._
+import io.circe.syntax._
+import io.circe.{Encoder, Printer}
+import org.apache.commons.compress.archivers.tar.{TarArchiveEntry, TarArchiveOutputStream}
+import org.apache.commons.compress.compressors.gzip.GzipCompressorOutputStream
+import org.apache.commons.io.output.ByteArrayOutputStream
+import org.docx4j.Docx4J
+import org.docx4j.openpackaging.packages.WordprocessingMLPackage
+import org.typelevel.log4cats.Logger
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+import software.amazon.awssdk.services.sqs.model.SendMessageResponse
+import uk.gov.nationalarchives.dp.client.ContentClient.{SearchField, SearchQuery}
+import uk.gov.nationalarchives.dp.client.Entities.Entity
+import uk.gov.nationalarchives.dp.client.EntityClient.StructuralObject
+import uk.gov.nationalarchives.dp.client.fs2.Fs2Client.{contentClient, entityClient}
+
+import java.security.MessageDigest
+import java.time.OffsetDateTime
+import java.util.UUID
+
+object Steps {
+  private val url = sys.env("PRESERVICA_API_URL")
+  private val secretName = sys.env("PRESERVICA_SECRET_NAME")
+  private val environment = sys.env.getOrElse("ENVIRONMENT", "intg")
```

Do we need a `.getOrElse` here? If there's no environment, wouldn't it be better to throw an error? Also, I was thinking if you're using the sandbox env, and you forget to add an env, maybe the error message might be confusing?

---

It's a fair point. It was easier when I was testing but you're right. I've taken it out.